src/app/components/leftbar/icons/body-icon/body-icon.component.scss

```diff
@@ -1,4 +1,10 @@
-.cls-1{fill:none;}
-.body {
-    fill: #000;
-}
\ No newline at end of file
+.ccf-body-icon-wrapper {
```

Comment:
Update selector to .ccf-body-icon.wrapper

Reply:
shouldnt this be .ccf-body-icon, .wrapper

Reply:
No, the classes on the top level div should be "ccf-body-icon wrapper" (not the space between them), then the correct selector becomes .ccf-body-icon.wrapper (no space between!)

Reply:
For reference check out the html and sccs of: https://github.com/hubmapconsortium/ccf-ui/tree/pbi-18-tissue-browser/src/app/components/tissues-browser-grid-item